redis/src/main/scala/zio/redis/ByteStream.scala

```diff
@@ -28,59 +27,63 @@ import java.nio.channels.{AsynchronousSocketChannel, Channel, CompletionHandler}
 private[redis] object ByteStream {
   trait Service {
     def read: Stream[IOException, Byte]
-    def write(chunk: Chunk[Byte]): IO[IOException, Unit]
+    def write(chunk: Chunk[Byte]): IO[IOException, Option[Unit]]
   }
 
-  lazy val default: ZLayer[Logging, RedisError.IOError, Has[ByteStream.Service]] =
-    ZLayer.succeed(RedisConfig.Default) ++ ZLayer.identity[Logging] >>> live
+  lazy val default: ZLayer[Any, RedisError.IOError, Service] =
+    ZLayer.succeed(RedisConfig.Default) >>> live
 
-  lazy val live: ZLayer[Logging with Has[RedisConfig], RedisError.IOError, Has[ByteStream.Service]] =
-    ZLayer.fromServiceManaged[RedisConfig, Logging, RedisError.IOError, Service] { config =>
-      connect(new InetSocketAddress(config.host, config.port))
+  lazy val live: ZLayer[RedisConfig, RedisError.IOError, Service] =
```

Comment:
Not work,I don't know why error shows connect is closed.

Reply:
Please investigate.

Reply:
resolved